.github/workflows/test.yml

```
@@ -32,26 +32,11 @@ jobs:
       #      tmp/
 
       - uses: DeterminateSystems/nix-installer-action@v1
```

Maybe it's time to update our installer action too?
```suggestion
      - uses: DeterminateSystems/nix-installer-action@v4
```